keps/prod-readiness/sig-apps/4368.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 4368
 alpha:
   approver: "@wojtek-t"
+beta:
+  approver: "@wojtek-t"
```

Comment:
Few comments: 

> We decide to keep the field immutable, at least for Alpha, we will re-evaluate the decision for Beta

I'm assuming we're leaving it immutable for Beta too - please reflect that.


> The Upgrade->downgrade->upgrade testing will be done manually prior to release as Beta, with the following steps:

Did that happen?

Reply:
> I'm assuming we're leaving it immutable for Beta too - please reflect that.

Yes, we want to keep it immutable, I discussed it within the community, and the decision was well received, so I even drop it from considerations for GA.

> Did that happen?

Cannot recall :) Just in case it didn't I repeated the steps and it worked fine. I also cleaned up the description slightly.